xml/admin_ceph_gateway.xml

```diff
@@ -3279,365 +3211,211 @@ rgw_zone=us-east-1</screen>
-&prompt.cephuser;<command>radosgw-admin</command> period pull --url=<replaceable>URL-TO-MASTER-ZONE-GATEWAY</replaceable> \
---access-key=<replaceable>ACCESS-KEY</replaceable> --secret=<replaceable>SECRET</replaceable>
-     </screen>
-    </step>
-    <step>
-     <para>
-      Make the recovered zone the master and default zone.
-     </para>
-<screen>
-&prompt.cephuser;<command>radosgw-admin</command> zone modify --rgw-zone=<replaceable>ZONE-NAME</replaceable> --master --default
-     </screen>
-    </step>
-    <step>
-     <para>
-      Update the period to make the changes take effect.
-     </para>
-<screen>
-&prompt.cephuser;<command>radosgw-admin</command> period update --commit
-     </screen>
-    </step>
-    <step>
-     <para>
-      Then, restart the Ceph Object Gateway in the recovered zone.
-     </para>
-<screen>
-&prompt.root;<command>systemctl</command> restart ceph-radosgw@rgw.`hostname -s`
-     </screen>
-    </step>
-    <step>
-     <para>
-      If the secondary zone needs to be a read-only configuration, update the
-      secondary zone.
-     </para>
-<screen>
-&prompt.cephuser;<command>radosgw-admin</command> zone modify --rgw-zone=<replaceable>ZONE-NAME</replaceable> --read-only
-     </screen>
-    </step>
-    <step>
-     <para>
-      Update the period to make the changes take effect.
-     </para>
-<screen>
-&prompt.cephuser;<command>radosgw-admin</command> period update --commit
-     </screen>
-    </step>
-    <step>
-     <para>
-      Finally, restart the Ceph Object Gateway in the secondary zone.
-     </para>
-<screen>
-&prompt.root;<command>systemctl</command> restart ceph-radosgw@rgw.`hostname -s`
-     </screen>
-    </step>
-   </procedure>
-  </sect2>
- </sect1>
+  <sect2 xml:id="ceph-rgw-maintenance">
+   <title>Maintenance</title>
+
+   <sect3 xml:id="ceph-rgw-check-sync">
+     <title>Checking the Sync Status</title>
+     <para>Information about the replication status of a zone can be
+       queried with:</para>
+<screen>&prompt.cephuser;radosgw-admin sync status
+        realm b3bc1c37-9c44-4b89-a03b-04c269bea5da (gold)
+    zonegroup f54f9b22-b4b6-4a0e-9211-fa6ac1693f49 (us)
+         zone adce11c9-b8ed-4a90-8bc5-3fc029ff0816 (us-west)
+        metadata sync syncing
+              full sync: 0/64 shards
+              incremental sync: 64/64 shards
+              metadata is behind on 1 shards
+              oldest incremental change not applied: 2017-03-22 10:20:00.0.881361s
+    data sync source: 341c2d81-4574-4d08-ab0f-5a2a7b168028 (us-east)
+                      syncing
+                      full sync: 0/128 shards
+                      incremental sync: 128/128 shards
+                      data is caught up with source
+              source: 3b5d1a3f-3f27-4e4a-8f34-6072d4bb1275 (us-3)
+                      syncing
+                      full sync: 0/128 shards
+                      incremental sync: 128/128 shards
+                      data is caught up with source</screen>
```

Comment:
this is also repeated from the last section. Although it kinda makes sense. This is the Maintenance section and something they'd want to look up. Not sure what to do, just worth a mention, happy for it to stay like this.